data/core/macros/abilities.cfg

```diff
@@ -117,6 +117,42 @@ Adjacent own units of lower level will do more damage in battle. When a unit adj
     [/leadership]
 #enddef
 
+#define ABILITY_LEADERSHIP_TARGET ID ATTACK
```

Comment:
Remove this. We won't be accepting a new macro that isn't even used. You are of course free to include this macro in your addon's code, if you need it there.